exec/java-exec/src/main/java/org/apache/drill/exec/physical/impl/aggregate/StreamingAggBatch.java

```diff
@@ -82,11 +82,18 @@
   private boolean specialBatchSent = false;
   private static final int SPECIAL_BATCH_COUNT = 1;
 
-  public StreamingAggBatch(StreamingAggregate popConfig, RecordBatch incoming, FragmentContext context) throws OutOfMemoryException {
+  public StreamingAggBatch(StreamingAggregate popConfig, RecordBatch incoming, FragmentContext context)
+      throws OutOfMemoryException {
     super(popConfig, context);
     this.incoming = incoming;
   }
 
+  public StreamingAggBatch(StreamingAggregate popConfig, RecordBatch incoming, FragmentContext context,
+                           final boolean buildSchema) throws OutOfMemoryException {
```

Comment:
I am not familiar with the entire inner implementation of StreamingAggBatch. @amansinha100 could you please explain why we need another buildSchema flag?